frontend/src/lib/components/Errors/ErrorDisplay.tsx

```diff
@@ -102,6 +135,14 @@ export function getExceptionPropertiesFrom(eventProperties: Record<string, any>)
             }
         }
     }
+    // exception autocapture sets $exception_list for chained exceptions.
+    // If it's not present, get this list from the sentry_exception
```

Comment:
feels like `sentry_exception` should have been just a `$exception_list` with a single item, so we don't need to do this (that's how sentry does btw).
https://develop.sentry.dev/sdk/event-payloads/types/

Reply:
all `$exception_type,`, and other props are part of the `sentry_exception` type so
$exception_list is a list of sentry_exception

Reply:
so about the first comment.
sentry does either:
```json
{
  "exception": {}
}
```

or

```json
{
  "exception": {
    "values": []
  }
}
```

this is technical debt since chained exceptions weren't a thing before or not implemented at least.
backend|frontend has to check if it's either a single exception or a list of exceptions based on the `values` being present or not.

Always being:

```json
{
  "exception": {
    "values": [its ok to be a single item here even if its not a chained exception]
  }
}
```

will reduce those unnecessary checks.

Reply:
in our case, we'd always use `$exception_list`, even if it is a single item, and forget about `sentry_exception`

Reply:
Yeah correct, sounds like we're aligned. I have this here for the case where we're extracting and displaying the exception from sentry, not one captured via posthog (posthog will always send a list)

Reply:
We probably still need this for backward compatibility but should we look to update the sentry integration to use `$exception_list` over `$sentry_exception`? Happy for that to come in another PR if you prefer

Reply:
I quite like keeping these two separate, since that ensures we don't have a hard dependency on sentry's schema - the sentry integration can always just get the sentry exception, in sentry's format, and then there's this translation layer that converts it to ours where need be. Doing it here means there's only one place we need to do the conversion (rather than in all SDKs that eventually might support this), and also we don't need to rush to update SDKs when sentry adds/removes things from their schema

Reply:
unless we are using a super old version of the sentry js SDK, the `event.exception` is always an object that contains a `values` list, so technically this workaround would not even be needed
https://github.com/getsentry/sentry-javascript/blob/8ce9f7ce9a122bfe70d3c12ebfe50bba5d237037/packages/utils/src/eventbuilder.ts#L150-L154

Reply:
hmm, the point is if our `$exception_list` diverges from sentry (like for example, the mechanism is set differently, handled is not a boolean), then we can do that translation from a sentry exception to a posthog exception here. But if we don't know that it came from a sentry exception, this becomes much more annoying, since you then need to test if mechanism is one of acceptable values, and if not, convert it, etc. etc.

This is just a translation layer from sentry exception to posthog exception, not a workaround, if I'm understanding this code correctly, so I'd always want this because of reasons in above comment 😅